docker-bake.hcl

```diff
@@ -269,7 +269,13 @@ group "all" {
 # All Rust services defined in src/rust/Dockerfile should inherit from
 # this target.
 target "_rust-base" {
-  context    = "src"
+  context = "src"
+
+  # Additional named contexts: 
+  # https://www.docker.com/blog/dockerfiles-now-support-multiple-build-contexts/
+  contexts = {
+    dist-ctx = "dist"
+  }
```

Comment:
tbh I like that it's named something slightly different than `dist` so that one can ctrl-f backtrack to figure out "what the hell is this `--from` about" but i'm not married to it